README.md

```diff
@@ -20,20 +21,23 @@
 	- [Exporting Particle Systems](#exporting-particle-systems)
 	- [Exporting Shape (or Pose) Animations](#exporting-shape-animations)
 	- [Exporting Node Animations](#exporting-node-animations)
+ 	- [Mesh Previewer](#mesh-previewer)
  - [About](#about)
  - [Authors](#authors)
 
 ## Installing
 1. Copy the [io_ogre](io_ogre) folder into the [$BLENDER_DIR](https://docs.blender.org/manual/en/latest/advanced/blender_directory_layout.html)`/scripts/addons` folder.
-  - Windows = %USERPROFILE%\AppData\Roaming\Blender Foundation\Blender\[VERSION]\scripts\addons
-
-2. Set the correct path to `OGRETOOLS_XML_CONVERTER` in [io_ogre/config.py](io_ogre/config.py) (Line 108) prior to the first run.
-  - If you want to export meshes for Ogre (v1): Then this path should point to `OgreXMLConverter.exe`. This file can be found in the [Ogre SDK](https://www.ogre3d.org/download/sdk/sdk-ogre)
-  - If you want to export meshes for OgreNext (v2.*): Then this path should point to `OgreMeshTool.exe`. This file can be found in the [OgreNext SDK](https://www.ogre3d.org/download/sdk/sdk-ogre-next)
-
+	- Windows = %USERPROFILE%\AppData\Roaming\Blender Foundation\Blender\[VERSION]\scripts\addons
+
+2. Configure the plugin by editing [io_ogre/config.py](io_ogre/config.py) prior to the first run.
```

Comment:
actually, you can set all these paths with the UI:
![paths](https://user-images.githubusercontent.com/4941802/133935949-33f2fcac-6c16-44b4-8829-2f8b898dcfa1.png)

Reply:
I know you can in theory. There is a possible weird bug where they don't save/load properly. It is on my todo list. I found the most reliable way was to just follow the original readme instructions.